### PR TITLE
Changelog 2024-06-03

### PR DESCRIPTION
**Pull request by tsg**
For past 2 weeks.

### CODE REVIEW

File: 2024-06-03.mdx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+date: 6-3-2024
+---
+
+### What's new with Xata
+
+**UI improvements**
+
+You might have noticed already, but we've been beautifying the UI in the past weeks. For example, check out this empty state page for a new database: 🤩
+
+![Empty state for the new database page](/images/2024-06-03-empty-state.png)
```
Comment (alexfrancoeur) [inline 2024-06-03.mdx:11]: We showcased this in the last changelog (3rd section down) https://github.com/xataio/mdx-changelog/blob/main/2024-05-22.mdx

I think the running multiple statements at once is actually a pretty big improvement in the playground UX -- our second most popular app behind the table. What do you think about a quick screen studio recording and leading with that? Happy to do it if you'd like.

File: 2024-06-03.mdx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  * Improved compatibility with more tools that use the Postgres wire protocol, including JetBrains DataGrip.
+  * Added support for several system info functions.
+  * Allow locking clauses in SELECT statements. For example, the following is now supported: `SELECT * FROM table1 WHERE name = 'John' FOR UPDATE`.
+  * Improved compatibility and fixed issues when running [Ghostfolio](https://github.com/ghostfolio/ghostfolio) with Xata.
+  * Reduced clutter and errors during pg_restore runs.
+  * Added support for `SHOW session_replication_role`.
+
+**Extensions!**
+
+We added support for the first extensions on shared clusters:
```
Comment (alexfrancoeur) [inline 2024-06-03.mdx:32]: Might be worth clarifying that this is only for the public beta (and a link to the docs?)

File: 2024-06-03.mdx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+* Added support limited support for DO blocks. More improvements coming soon.
+
+**Data API**
+
+Also our API, more specifically the SQL over HTTP part, got an important update:
+
+* Support batching multiple SQL commands in a single request over the HTTP API.
+
+
+### Blog posts
```
Comment (alexfrancoeur) [inline 2024-06-03.mdx:52]: Depending on when this ships, we may have a tutorial from Teri tomorrow according to our content schedule. @CezzaineZ let us know if we need to update the changelog, no reason to rush if it's not there though.

**Review by alexfrancoeur — APPROVED** [review]

File: 2024-06-03.mdx (hunk 1 above)
Comment (tsg) [reply 2024-06-03.mdx]: I did this. The video is not working in preview, but it should work once merged.

**Comment by alexfrancoeur** [comment]
LGTM, Dave had suggested I store the video in the Xata DB instead of GitHub with my last one (https://github.com/xataio/mdx-changelog/pull/36)

> https://app.xata.io/workspaces/xata-uq2d57/dbs/website:us-east-1/branches/main/tables/public-files
> you can use this table to host your images
> so you don't need to deal with waiting on github to publish for staging

File: 2024-06-03.mdx [hunk 4]
```diff
@@ -0,0 +1,58 @@
+---
+date: 6-3-2024
+---
+
+### What's new with Xata
+
+**UI improvements**
+
+The UI Playground now supports running multiple statements at once. This was a highly requested feature that will make it easier to run scripts in the Playground.
```
Comment (SferaDev) [inline 2024-06-03.mdx:9]:
```suggestion
The UI Playground now supports running multiple SQL statements at once. This was a highly requested feature that will make it easier to run scripts in the Playground.
```

**Comment by alexfrancoeur** [comment]
@tsg anything blocking us (besides the build) from merging and doing social / discord?

**Comment by tsg** [comment]
@alexfrancoeur nope, I think it's good to go now. Merging.

**Pull request merged**